### DISCUSSION TITLE
/infra/control/main 哪里去了？

### DISCUSSION THREAD

**Discussion by dickfej** [v2fly/v2ray-core#1735] [General]
编译v2ctl发现/infra/control/main不见了，无法编译，请问是移动到别的地方还是怎么回事？

**Comment by AkinoKaede** [comment]
It has been removed in v5.

**Reply by lutcraft** [reply]
v5分支么，我在里面也没用找到这个目录

**Reply by lutcraft** [reply]
v5是还没做这一部分么，我看v4里还有

**Reply by Remonli** [reply]
v5 取消这个文件了

**Reply by lutcraft** [reply]
那在v5我应该如何编译v2ctl呢

**Reply by Remonli** [reply]
没有 v2ctl这个东西了。

**Reply by AkinoKaede** [reply]
`v2ctl` has been remove. #488

**Reply by lutcraft** [reply]
so the functions of the former v2ctl are provided by v2ray, which is runed both on the my pc and foreign server now?
